clients/java/src/main/java/io/camunda/client/api/search/response/SearchResponsePage.java

```diff
@@ -20,6 +20,23 @@ public interface SearchResponsePage {
   /** Total number of items that matches the query */
   Long totalItems();
 
+  /**
+   * Whether more than 10,000 results exist in Elasticsearch (ES) or OpenSearch (OS) searches.
+   *
+   * <p>In ES or OS, total hits are capped at 10,000. If the result set is greater than or equal to
```

Comment:
💭 Strictly speaking, this description is not precise. The 10k limit is just the default. It can be overridden in the ES/OS configuration. So, we might want to remove the reference to 10k items here.